File: PATTERNS/Pattern14/Pattern14.c
// *****
//  ****
//   ***
//    **
//     *

#include <stdio.h>

void main()
{
    int n;
    printf("Enter the row number:");
    scanf("%d", &n);

    for (int i = 1; i <= n; i++)
    {
        for (int k = 1; k <= i; k++)
        {
            printf(" ");
        }
        for (int j = i; j <= n; j++)
        {
            printf("*");
        }
        printf("\n");
    }
}

File: PATTERNS/Pattern14/Pattern14.go
// *****
//  ****
//   ***
//    **
//     *

package main

import "fmt"

func main() {
	var n int
	fmt.Print("Enter the row number:")
	fmt.Scanf("%d", &n)

	for i := 1; i <= n; i++ {
		for k := 1; k <= i; k++ {
			fmt.Print(" ")
		}
		for j := i; j <= n; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}
